pkg/vm/airdrop.go

```diff
@@ -13,6 +13,12 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
+const (
+	defaultAirdrop        = "Airdrop 1 million tokens to the default address (do not use in production)"
+	customAirdrop  string = "Customize your airdrop"
```

Comment:
type seems not needed

Reply:
My editor was complaining about a type checking thing, but I think it fixed itself so I removed this.